### PR TITLE
Add py.typed to mark the library as typed

### PR DESCRIPTION
**Pull request by davep**
Removes an error like this:

    error: Skipping analyzing "ollama":
    module is installed, but missing library stubs or py.typed marker  [import-untyped]

when running mypy over code that uses the library.

### CODE REVIEW

**Comment by bpshaver** [comment]
@jmorganca Any updates on this? It is quite hard to tell who the maintainer of this project is.

**Comment by bpshaver** [comment]
I don't use poetry but I would be very surprised if it doesn't automatically include `py.typed` in the distribution package.

**Comment by davep** [comment]
@bpshaver You may be right, but I remember us needing to do this to convince Poetry to include it when building Textual; perhaps that's changed in the last year or so?

**Comment by davep** [comment]
Just did a test build without the `include` and it does look like it does include it anyway.

**Review by JonZeolla — APPROVED** [review]

**Comment by mxyng** [comment]
@davep I'm happy to merge this if you can the pyproject.toml change since poetry includes it

**Comment by davep** [comment]
If I can? Did a word get missed? Sorry, don't quite understand what's being asked.

**Comment by mxyng** [comment]
I missed a word "remove"

**Comment by davep** [comment]
Ahh! Got you! Sure, I'll sort that when I'm next at a keyboard (hopefully within the next hour).

**Comment by davep** [comment]
@mxyng Done!

**Review by mxyng — APPROVED** [review]

**Pull request merged**
